custom-images/generate_custom_image.py

```diff
@@ -378,6 +355,16 @@ def perform_sanity_checks(args):
     raise Exception("Invalid path to customization script: '{}' is not a file.".format(
         args.customization_script))
 
+  # Check the image doesn't already exist.
+  command = "gcloud compute images describe {} --project={}".format(
+      args.image_name, args.project_id)
+  with open(os.devnull, 'w') as devnull:
+    pipe = subprocess.Popen(
+        [command], stdout=devnull, stderr=devnull, shell=True)
```

Comment:
Is `devnull` a valid value?

Reply:
It is created at line 361 `with open(os.devnull, 'w') as devnull:`